app/src/main/java/org/thoughtcrime/securesms/ui/components/Button.kt

```diff
@@ -291,16 +114,57 @@ fun BorderlessHtmlButton(
         Text(
             text = annotatedStringResource(textId),
             textAlign = TextAlign.Center,
-            style = MaterialTheme.typography.extraSmall,
+            style = extraSmall,
             modifier = Modifier.padding(horizontal = 2.dp)
         )
     }
 }
 
-private val MutableInteractionSource.releases
-    get() = interactions.filter { it is PressInteraction.Release }
-
 @Composable
-fun SmallButtons(content: @Composable () -> Unit) {
-    CompositionLocalProvider(LocalButtonSize provides smallButton) { content() }
+fun NotificationRadioButton(
```

Comment:
Is there a reason this was moved here if it is only used in that one screen?
Was the idea  instead to make a generic radio button component, that the notification one would set with its specific text?

Reply:
Just that its kinda self-contained component, maybe it could be its own `NotificationRadioButton` and stay in a package near its usage, or something with a better name, as the component could be reused...

Reply:
If something is only used once, you can have it declared as its own composable in the file ( screen ) where it exists.
If you are building a reusable component, then in this case it should be a generic, Session styled, radio button, which looks like this notification one but isn't tied to it.
If we are not reusing these buttons anywhere, then it's in the realm of the `MessageNotificationsScreen`

Reply:
@bemusementpark

Reply:
cool. moved it back.